src/test/java/ca/qaguru/oranghrmbatch27/pages/LicensePage.java

```diff
@@ -50,4 +53,22 @@ public void saveNewLicense(String license)
         System.out.println(license + " is added successfully");
         System.out.println("\n");
     }
+    // Method to Edit License
+    public void editLicense(String oldlicense, String newText)
+    {
+       click(By.xpath("//div[text()='"+oldlicense+"']/ancestor::div[@class='oxd-table-card']/descendant::i[3]"));
+       setText(By.cssSelector(idLicTxtLevel),newText);
+        if (getText(By.xpath(lblAlreadyExistsMessage)).contains("Already exists")) {
+            click(By.xpath(getIdLicCancelBtn));
+        } else {
+            click(By.xpath(idLicSaveBtn));
+        }
+        isElementVisible(By.cssSelector(tblLicense));
+        for (WebElement License : listLicense)
+        {
+            String txtLicense = License.getText();
+            System.out.println(txtLicense);
```

Comment:
bool isSaved = false;
for (WebElement License : listLicense)
        {
            String txtLicense = License.getText();
            if(txtLicense.equals(newText)){
              isSaved = true;
              break;
}
Assert.assertTrue(isSaved);


            
}